backendAPI/views.py

```diff
@@ -8,7 +8,7 @@ def check_login(request):
     try:
         user = User.objects.get(username=request.user.username)
         user_id = user.id
-        username = ''
+        username = 'Kafka'
```

Comment:
Do we need to keep the default user name as this? Because in the open link sharing case, all users will be given the same user name and it might be confusing. For now can we do something simple like `User1`, `User2` etc. ?